CMakeLists.txt

```diff
@@ -808,14 +808,6 @@ if(FUZZ)
   add_subdirectory(fuzz)
 endif()
 
-if(RUST_BINDINGS)
```

Comment:
Why remove this from CMake? Some of the CMake logic seems to be moved to bash scripts.

Reply:
We might be able to reintroduce a CMake mechanism to produce Rust bindings for someone's local use, but that would have little value for consumers. The Rust community needs a way to utilize AWS-LC by simply listing a crate/version in their Cargo.toml. The goal of this PR is to generate an "aws-lc-sys" Rust crate that can be published for that purpose.